Jenkinsfile

```diff
@@ -44,4 +56,34 @@ pipeline {
       }
     }
   }
+  post {
```

Comment:
Can we refactor the steps below?

```suggestion
post {
  when {
    GIT_BRANCH == 'latest'
   }
 }
```

Disclaimer: haven't tested this, but [should work in theory](https://jenkins.io/doc/book/pipeline/syntax/#when).

Reply:
Shame! The only other thought is to refactor with closures:

```
def onLatestOnly(callback) {
  if (GIT_BRANCH == 'latest') {
    callback()
  }
}
```

Then lower down we'd:

```
onLatestOnly(notifySlack('danger', 'Aborted', gitCommitAuthor, stageName, gitCommitMessage, slackChannel))
```

Reply:
Would be useful on line 43 too!